src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -110,8 +110,12 @@ class ProfileView extends React.Component {
               <ProfileTOC militaryInformation={militaryInformation} />
               <div id="contact-information" />
               <ContactInformation />
-              <div id="direct-deposit" />
-              <PaymentInformation />
+              {featureFlags.directDeposit && (
```

Comment:
We are now gating the display of the PaymentInformation component here rather than in the PaymentInformation component itself